src/search_count_report.py

```diff
@@ -0,0 +1,46 @@
+import csv
+from datetime import datetime
+
+from src.report import Report
+import src.utils as utils
+
+class SearchCountReport(Report):
+
+  def generate(self):
+    # Write fields to search_count.csv 
+    self.writer.writerow(['day', 'hour', 'count'])
+
+    time_to_count_dict = {}
```

Comment:
should we rename this to `day_hour_to_count_dict`

Reply:
i think `time_to_count_dict` is clearer